### ISSUE TITLE
In the lyrics panel it says Python was not found...

### ISSUE THREAD

**Issue by CloudyCrayons** [lukaszsulek/get_lyric#2]
Full error is "Python was not found; run without arguments to install from the Microsoft Store, or disable this shortcut from Settings > Manage App Execution Aliases.:

I successfully used the test script (.\get_lyric.py --artist "Trace Mountains" --song "Absurdity) and that worked but whenever I view the lyrics in the box itself the "lyrics" are just that error.

Because your test script worked it might not be a problem on your end but maybe somewhere else along the chain? not sure.

**Comment by CloudyCrayons** [comment]
I am using portable installation if that changes anything...

**Comment by lukaszsulek** [comment]
Hey there! Sorry for the delay! Did you manage to sort it out? If not, it could be a number of things. My first hunch would be to check if your pathing is setup correctly for python. Check the below stackoverflow link out.

https://stackoverflow.com/questions/13596505/python-not-working-in-command-prompt

**Comment by CloudyCrayons** [comment]
Sorry for late response. 

I installed Python 32 bit and re-installed 64 bit while also making sure I checked adding stuff to variables. This still does not fix my issue,  
![image](https://user-images.githubusercontent.com/32446731/119314557-e291c700-bc42-11eb-91d4-60275e42e711.png)
![ApplicationFrameHost_WaVZTcPG4m](https://user-images.githubusercontent.com/32446731/119315280-ae6ad600-bc43-11eb-8817-26475889114f.png)
![SystemPropertiesAdvanced_O7rxQ5WuCD](https://user-images.githubusercontent.com/32446731/119315286-b0349980-bc43-11eb-95f3-7b076946b9b4.png)

I have one song without a lyrics.txt file and when I tried to play it I got an error log file so you might be able to find something out more than me. One weird thing on line 431 is that it was trying to write something to the foobar installation on my desktop. however, I don't have my portable version of foobar on my desktop anymore. I moved it to my C Drive. Is it possible one of the plugins think my installation is in a different folder?? This error only happened once and I can't recreate it.

**Comment by CloudyCrayons** [comment]
[failure_00000001.txt](https://github.com/lukaszsulek/get_lyric/files/6530906/failure_00000001.txt)

**Comment by lukaszsulek** [comment]
Sorry, just seeing this now @CloudyCrayons ! From what I can tell, I don't believe you have the lyric plugin and my script set up and/or configured correctly. Can you share a screenshot of how you have the `Lyric Show 3` plugin configured? It's located under `Preferences -> Tools -> Lyric Show 3`. My README has a screenshot of how it should look like, and also please confirm you followed the other steps of placing the script in the right location.

**Comment by CloudyCrayons** [comment]
Ok after doing more troubleshooting I fixed the error. I did the steps from the start it ended up working. I don't really know what part I was doing wrong. (sorry if anyone else has this problem looking for my solution.)

_I decided to get a different theme anyway for foobar that doesnt use Show Lyric 3 it uses ESLyric which I think is just worse? lol.
Really hate how many different lyric standards there are. 

Anyway I really appreciate you sticking with me and helping me troubleshoot this stuff._
